src/pathwayvis/components/map/map.component.ts

```diff
@@ -482,9 +482,11 @@ class MapComponentCtrl {
     d3.selectAll('.reaction, .reaction-label').on('mouseenter', (d) => {
       const currentReaction = reactions.find((reaction) => reaction.id === d.bigg_id);
       const tooltipContainer = d3.select('div#tooltip-container');
-      tooltipContainer.select('#upperbound').property("value", currentReaction.upper_bound);
-      tooltipContainer.select('#lowerbound').property("value", currentReaction.lower_bound);
-        this._getContext();
+      if (tooltipContainer) {
```

Comment:
According to [this](https://sentry.io/technical-university-of-denmark/decaf-metabolica-ui-staging/issues/531783566/?referrer=slack) `currentReaction` was undefined.